### DISCUSSION TITLE
Button to refresh models for XY

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#237] [Ideas]
When I want to set some models for XY, SM helps showing my models below XY.
But the list is static, I haven't found a way to refresh it(except reloading the page) :-(
Could you please add a button to refresh the list?
![изображение](https://github.com/hako-mikan/sd-webui-supermerger/assets/1073832/6ef9ec4c-f190-42e1-a614-601f1bbe617d)

**Comment by wkpark** [comment]
it could be replaced by ~~`gr.Dropdown(multiselect=True,...)`~~ with `create_refresh_button()`

**Reply by wkpark** [reply]
https://github.com/hako-mikan/sd-webui-supermerger/pull/233/commits/d5decbb2e875f90bbae27358def2cf3c256a216d#diff-b28b40d126cec8f16450ec381d02337dc0791c82723f4fd86cdd907f5624d7daR203
refresh button added
